js/functions.js

```diff
@@ -89,7 +89,7 @@ function getRequest(url, handler)
 
 	request.onerror = function()
 	{
-		console.log('Error: ' + error);
+		console.log('Error: ' + request.status);
```

Comment:
squash

Reply:
Squash with which commit?

Reply:
Seems like an older bug, so keep it as is.